src/hooks/core/websocket/types.d.ts

```diff
@@ -111,6 +114,7 @@ export type GetstreamWebsocket = {
   unread_channels: number;
   unread_count: number;
   targetName: string;
+  reply_data?: any;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, the `reply_data` property is of type `any`. It's better to use a more specific type.
```diff
-  reply_data?: any;
+  reply_data?: ReplyDataType; // Replace ReplyDataType with the actual type
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->